ndbench-geode-plugins/src/main/java/com/netflix/ndbench/geode/plugin/EnvParser.java

```diff
@@ -0,0 +1,91 @@
+package com.netflix.ndbench.geode.plugin;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Reused from
+ * https://github.com/gemfire/cf-gemfire-connector-examples
+ *
+ * @author Pulkit Chandra
+ */
+public class EnvParser {
```

Comment:
What is the goal of this class? Is it just for reading the environment variables? if so, please use https://github.com/Netflix/archaius